src/TagInput/driver.js

```diff
@@ -0,0 +1,31 @@
+export default class TagInputDriver
+{
+    constructor( wrapper )
+    {
+        this.wrapper = wrapper;
+    }
+
+    click()
+    {
+        this.wrapper.find( 'Tag' ).first().simulate( 'click' );
```

Comment:
Hmm the TagInput implementation doesn’t have a concept of `click`, only `clickClose`... so I don’t think this driver method is needed